net.shadps4.shadPS4.yaml

```diff
@@ -0,0 +1,85 @@
+app-id: net.shadps4.shadPS4
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: shadps4
+
+finish-args:
+  # gamepad and 3D hardware support
+  - --device=dri
+  - --device=input
```

Comment:
device=input is not allowed. it is not available in any stable release.

Reply:
is there an alternative that is allowed here? i dont think a controller works without at least `--device=input`

Reply:
no stable version of flatpak accept `--devic e=input`

Reply:
is there an alternative that is allowed here?